### PR TITLE
Stracciatella kernel

### PR DESCRIPTION
**Pull request by Etn40ff**
This is my take on the default kernel for both rm1 and rm2.

Sources are exactly those provided by RemarkableAS.

Configs differs from theirs as follows:
- Removed version name modifications
- UINPUT=m
- TUN=m
- PERF_EVENTS=y

It requeres kernelctl and it should be a safe kernel to test its
functionalities.

### CODE REVIEW

File: package/linux-stracciatella/package
Comment (gbyl) [inline package/linux-stracciatella/package]:
```suggestion
pkgdesc="RemarkableAS's vanilla kernel with a few extra flakes"
```

File: package/linux-stracciatella/package
Comment (gbyl) [inline package/linux-stracciatella/package]:
```suggestion
    echo -e "Please use kernelctl to select the kernel to boot."
```

**Review by gbyl — CHANGES_REQUESTED** [review]

**Comment by Etn40ff** [comment]
This kernel requires firmware blobs that were introduced in version 2.9. Now we print a warning when installing on an older version.

**Comment by luvanir** [comment]
RM1 owner here. linux-stracciatella works wonders! The only caveat is that you need to have at least version 2.9 of the remarkable OS before you switch away from the vanilla kernel. Dear maintainers, let me know if there are any tests you would like me to run before merging. Thank you for the help troubleshooting, Salvatore.

**Comment by Eeems** [comment]
@Etn40ff did you test this on a rM2?

**Comment by Eeems** [comment]
Tested on my rM1 as well and this works. I'd like someone to confirm that they tested on a rM2 and that it works.

**Comment by Etn40ff** [comment]
> @Etn40ff did you test this on a rM2?

It has been my daily drive since April.

**Review by Eeems — APPROVED** [review]

**Pull request merged**
